.gitignore

```diff
@@ -26,3 +26,5 @@ yarn-error.log*
 .vercel
 _ignore
 .idea
+package.json
```

Comment:
@dustinlarimer, should we be ignoring these files?

Reply:
@diana-macrometa `package.json` should _not_ be ignored.

Reply:
@dlozina-macrometa, please fix this. I'm betting the yarn file that follows also shouldn't be ignored.

Reply:
> @dlozina-macrometa, please fix this. I'm betting the yarn file that follows also shouldn't be ignored.

Fixed this to remove both files from .gitignore